tests/migration_test.go

```diff
@@ -2582,14 +2614,17 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					Skip("Skip DataVolume tests when CDI is not present")
 				}
 
-				quantity, err := resource.ParseQuantity(cd.FedoraVolumeSize)
-				Expect(err).ToNot(HaveOccurred())
+				sc, foundSC := libstorage.GetBlockStorageClass(k8sv1.ReadWriteMany)
```

Comment:
`exists` was used previously instead of `foundSC`, keep it the same here?

Reply:
I find it more clear like this. exists sometimes used for map value. Here we want to check if a specific storage class is found in the cluster.